homeassistant/components/media_player/squeezebox.py

```diff
@@ -303,3 +304,62 @@ def turn_on(self):
         """Turn the media player on."""
         self._lms.query(self._id, 'power', '1')
         self.update_ha_state()
+
+    def play_media(self, media_type, media_id, **kwargs):
+        """
+        Send the play_media command to the media player.
+
+        If ATTR_MEDIA_ENQUEUE is True, add `media_id` to the current playlist.
+        """
+        if kwargs.get(ATTR_MEDIA_ENQUEUE):
+            self.add_uri_to_playlist(media_id)
+        else:
+            self.play_uri(media_id)
+
+    def play_uri(self, media_id):
+        """
+        Replace the current play list with the uri.
+
+        <playerid> playlist play <item> <title> <fadeInSecs>
+
+        The "playlist play" command puts the specified song URL,
```

Comment:
This documentation is not for the user, it is for any future developer. I has always tried to add as much information in the code so future developers can work better with it. Is this not a stance for Home Assistant?